# Customize parameters for move_base
# Decrease max velocity
# For other parameters, indent with 2 spaces
move_base:
  DWAPlannerROS:
    max_vel_x: 0.17  # 0.55
    acc_lim_x: 1.0
    xy_goal_tolerance: 0.2  # 0.10
  
  local_costmap:
    inflation_layer:
      cost_scaling_factor: 20
      inflation_radius: 0.2
  
  planner_frequency: 0.0
  max_planning_retries: 1

navigation_velocity_smoother:
  speed_lim_v: 0.17
  accel_lim_v: 0.1
